### ISSUE TITLE
Acfutils build fails at "TEST_GLSL_VERSION" and "GLEW_ARB_gl_spirv"

### ISSUE THREAD

**Issue by JT8D-17** [skiselkov/libacfutils#14]
After applying bandaids to much of the dependency build process and much of the libacfutils build process (it's been hell on earth so far), I'm stuck.

> ../src/shader.c: In function ‘shader_from_spirv_fallback’:
../src/shader.c:135:27: error: ‘GLEW_VERSION_4_6’ undeclared (first use in this function); did you mean ‘GLEW_VERSION_4_3’?
  135 |         TEST_GLSL_VERSION(GLEW_VERSION_4_6, "glsl460");
      |                           ^~~~~~~~~~~~~~~~
../src/shader.c:126:33: note: in definition of macro ‘TEST_GLSL_VERSION’
  126 |                 if (!loaded && (version)) { \
      |                                 ^~~~~~~
../src/shader.c:135:27: note: each undeclared identifier is reported only once for each function it appears in
  135 |         TEST_GLSL_VERSION(GLEW_VERSION_4_6, "glsl460");
      |                           ^~~~~~~~~~~~~~~~
../src/shader.c:126:33: note: in definition of macro ‘TEST_GLSL_VERSION’
  126 |                 if (!loaded && (version)) { \
      |                                 ^~~~~~~
../src/shader.c: In function ‘shader_from_spirv’:
../src/shader.c:237:14: error: ‘GLEW_ARB_gl_spirv’ undeclared (first use in this function)
  237 |         if (!GLEW_ARB_gl_spirv ||
      |              ^~~~~~~~~~~~~~~~~
../src/shader.c:238:40: error: ‘GL_SHADER_BINARY_FORMAT_SPIR_V’ undeclared (first use in this function); did you mean ‘GL_SHADER_BINARY_FORMATS’?
  238 |             !have_shader_binary_format(GL_SHADER_BINARY_FORMAT_SPIR_V) ||
      |                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                        GL_SHADER_BINARY_FORMATS
../src/shader.c:280:9: error: implicit declaration of function ‘glSpecializeShader’ [-Werror=implicit-function-declaration]
  280 |         glSpecializeShader(shader, entry_pt, n_spec, spec_indices, spec_values);
      |         ^~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [Makefile.Debug:1362: debug/shader.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/media/Data/Git_Repos/libacfutils/qmake'
make: *** [Makefile:45: debug] Error 2

The header file chain does eventually point to `GL/glew.h` as far as I can see and I've pointed `qmake/qmake.pro` to `INCLUDEPATH += ../glew/glew-1.13.0-linux-64/include` and applied that potential fix from another issue, i.e. `DEFINES += GLEW_STATIC GLEW_MX`, but no success.

All I want is to compile a more up to date release of librain...
